packages/snaps-controllers/vitest.config.mts

```diff
@@ -0,0 +1,110 @@
+// eslint-disable-next-line import-x/no-nodejs-modules
+import { join } from 'path';
+import { nodePolyfills } from 'vite-plugin-node-polyfills';
+import tsconfigPaths from 'vite-tsconfig-paths';
+import { defineConfig } from 'vitest/config';
+
+const IFRAME_PATH = join(
+  import.meta.dirname,
+  '../snaps-execution-environments/dist/browserify/iframe',
+);
+
+const IFRAME_TEST_PATH = join(
+  import.meta.dirname,
+  './src/services/iframe/test',
+);
+
+const WORKER_EXECUTOR_PATH = join(
+  import.meta.dirname,
+  '../snaps-execution-environments/dist/browserify/worker-executor',
+);
+
+const WORKER_POOL_PATH = join(
+  import.meta.dirname,
+  '../snaps-execution-environments/dist/browserify/worker-pool',
+);
+
+export default defineConfig({
+  plugins: [tsconfigPaths(), nodePolyfills()],
+
+  server: {
+    proxy: {
+      '/iframe/executor': {
+        target: `http://localhost:63315/@fs${IFRAME_PATH}`,
+        rewrite: (path) => path.replace(/^\/iframe\/executor/u, ''),
+      },
+
+      '/iframe/test': {
+        target: `http://localhost:63315/@fs${IFRAME_TEST_PATH}`,
+        rewrite: (path) => path.replace(/^\/iframe\/test/u, ''),
+      },
+
+      '/worker/executor': {
+        target: `http://localhost:63315/@fs${WORKER_EXECUTOR_PATH}`,
+        rewrite: (path) => path.replace(/^\/worker\/executor/u, ''),
+      },
+
+      '/worker/pool': {
+        target: `http://localhost:63315/@fs${WORKER_POOL_PATH}`,
+        rewrite: (path) => path.replace(/^\/worker\/pool/u, ''),
+      },
+    },
+
+    fs: {
+      strict: true,
+      allow: [
+        './src/services/iframe/test',
+        '../snaps-execution-environments/dist/browserify/iframe',
+        '../snaps-execution-environments/dist/browserify/worker-executor',
+        '../snaps-execution-environments/dist/browserify/worker-pool',
+      ],
+    },
+  },
+
+  test: {
+    // Vitest enables watch mode by default. We disable it here, so it can be
```

Comment:
Wondering if we should set `maxWorkers` for Chrome too? IIRC WDIO was super slow with too many workers

Reply:
Performance seems fine with the default setting. It's only required for Firefox because of a Playwright bug.